package.json

```diff
@@ -59,12 +59,14 @@
     "enzyme": "^2.8.2",
     "eslint": "^3.17.1",
     "eslint-config-wordpress": "^1.1.0",
+    "eslint-plugin-jest": "20.0.3",
     "eslint-plugin-jsx-a11y": "^4.0.0",
     "eslint-plugin-react": "^6.10.3",
     "expose-loader": "^0.7.3",
     "extract-text-webpack-plugin": "^2.1.0",
     "gettext-parser": "^1.2.2",
     "glob": "^7.1.1",
+    "jest": "20.0.4",
```

Comment:
I think we should use `^` to include all non-blocking updates, I guess it's also the default when using npm install --save